### PR TITLE
Update output formatting

### PR DESCRIPTION
**Pull request by chrisberthe**
Updates the `theme-lint` reporter and modifies the formatting to coincide with what we're currently using as part of `slate-tools`.

image

### CODE REVIEW

File: reporter.js [hunk 1]
```diff
@@ -11,37 +12,41 @@ module.exports = class Reporter {
...
     if (this.failures.length === 0) {
       this.outputStream.write([
-        chalk.green('All good!'),
-        `(${testsRun} checks run)`
+        chalk.green(`Success (${testsRun} checks run)`)
+      ].join(' ') + '\n\n');
+    } else {
+      this.outputStream.write([
+        chalk.red(`Failed (${testsRun} checks run)`)
       ].join(' ') + '\n\n');
```
Comment (davidcornu) [inline reporter.js]: These 3 lines can be condensed to

```js
this.outputStream.write(chalk.red(`Failed (${testsRun} checks run)`) + '\n\n');
```

File: reporter.js [hunk 2]
```diff
@@ -11,37 +12,41 @@ module.exports = class Reporter {
...
     const testsRun = this.failures.length + this.successes.length;
 
+    this.outputStream.write('Translation tests complete: ');
+
     if (this.failures.length === 0) {
       this.outputStream.write([
-        chalk.green('All good!'),
-        `(${testsRun} checks run)`
+        chalk.green(`Success (${testsRun} checks run)`)
+      ].join(' ') + '\n\n');
```
Comment (davidcornu) [inline reporter.js]: These 3 lines can be condensed to

```js
this.outputStream.write(chalk.green(`Success (${testsRun} checks run)`) + '\n\n')
```
```

File: reporter.js [hunk 3]
```diff
@@ -11,37 +12,41 @@ module.exports = class Reporter {
...
+        this.outputStream.write(chalk.red(`${file}:\n`));
+
+        failures.map((failure) => {
+          return this.outputStream.write(`${failure[0]}\n`);
+        });
+      });
+
+      this.outputStream.write('\n');
+
+      throw new Error('Translation errors detected.');
```
Comment (davidcornu) [inline reporter.js]: This breaks the contract for `Reporter`. The exit code is handled by `cli.js`.

File: reporter.js [hunk 4]
```diff
@@ -11,37 +12,41 @@ module.exports = class Reporter {
...
+    }
+
+    this.successes = this.failures = [];
```
Comment (davidcornu) [inline reporter.js]: Why? We initialize a new reporter for every test run. This shouldn't be needed and breaks the contract with `cli.js`.

**Comment by davidcornu** [comment]
A lot of the proposed changes are breaking. We should make sure this is a major version bump.

**Comment by chrisberthe** [comment]
Thanks for the review - was planning on bumping the version in a future PR.

**Comment by chrisberthe** [comment]
Good for another round of reviews.

**Pull request merged**